tests/general/QuotesBearTest.py

```diff
@@ -66,34 +66,32 @@ def test_valid_quotes(self):
                           dependency_results=self.dep_results) as results:
             self.assertEqual(len(results), 0)
 
+        self.uut.run('F', self.single_quote_file, self.dep_results, "'", True)
+
+        self.uut.run('F', self.single_quote_file, self.dep_results, "'", False)
+
     def test_invalid_quotes(self):
         with execute_bear(self.uut, self.filename, self.single_quote_file,
                           dependency_results=self.dep_results) as results:
             res_list = list(results)
             self.assertEqual(len(res_list), 1)
-            self.assertEqual(res_list[0].diffs[self.filename].unified_diff,
-                             '--- \n'
-                             '+++ \n'
-                             '@@ -2,5 +2,5 @@\n'
-                             " '''\n"
-                             ' Multiline string\n'
-                             " '''\n"
-                             "-'a string with single quotes!'\n"
-                             '+"a string with single quotes!"\n'
-                             ' "A double quoted string with \' in it"\n')
+            self.assertEqual(res_list[0].diffs[self.filename]._file,
+                             ['\n',
+                              "'''\n",
+                              'Multiline string\n',
+                              "'''\n",
+                              "'a string with single quotes!'\n",
+                              '"A double quoted string with \' in it"\n'])
 
         self.section['preferred_quotation'] = "'"
         with execute_bear(self.uut, self.filename, self.double_quote_file,
                           dependency_results=self.dep_results) as results:
             res_list = list(results)
             self.assertEqual(len(res_list), 1)
-            self.assertEqual(res_list[0].diffs[self.filename].unified_diff,
-                             '--- \n'
-                             '+++ \n'
-                             '@@ -2,5 +2,5 @@\n'
-                             " '''\n"
-                             ' Multiline string\n'
-                             " '''\n"
-                             '-"a string with double quotes!"\n'
-                             "+'a string with double quotes!'\n"
-                             " 'A single quoted string with \" in it'\n")
+            self.assertEqual(res_list[0].diffs[self.filename]._file,
```

Comment:
hmm not necessary to change, also `_file` is not part of the public API, what makes tests implementation-detail-dependent. Please use the old `unified_diff` attribute.
Same above.

Reply:
`unified_diff` theres no attribute named this. This is what the error shows up.And also `pprint(vars())` also shows no attribute named `unified_diff`.

Reply:
> `res_list[0].diffs[self.filename].unified_diff`

This was written before :O